packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -121,3 +161,69 @@ export function createVerifiedAttributeLogic({
 
   return toCreateEventAttributeAdded(newVerifiedAttribute);
 }
+
+export function createCertifiedAttributeLogic({
+  attribute,
+  apiCertifiedAttributeSeed,
+  certifier,
+}: {
+  attribute: WithMetadata<Attribute> | undefined;
+  apiCertifiedAttributeSeed: ApiCertifiedAttributeSeed;
+  certifier: string;
+}): CreateEvent<AttributeEvent> {
+  if (attribute) {
+    throw attributeDuplicate(apiCertifiedAttributeSeed.name);
+  }
+
+  const newCertifiedAttribute: Attribute = {
+    id: uuidv4(),
+    kind: attributeKind.certified,
+    name: apiCertifiedAttributeSeed.name,
+    description: apiCertifiedAttributeSeed.description,
+    creationTime: new Date(),
+    code: undefined,
```

Comment:
certified attributes should always have the `code` (it's the one passed in the request)